cli/kata-env.go

```diff
@@ -223,9 +222,6 @@ func getHostInfo() (HostInfo, error) {
 	}
 
 	hostVMContainerCapable := true
-	if runtim.GOARCH == "ppc64le" {
```

Comment:
We don't need to check other values, such as "ppc" ?

Reply:
No. Containers are only for ppc64le (PowerPC LE) platform.